### DISCUSSION TITLE
Code consistency / Workspace settings / Plugins

### DISCUSSION THREAD

**Discussion by nyandika** [reactdeveloperske/reactdevske-website#62] [General]
I am a single-quote person and so far in the code I am seeing double quotes being the trend. This has had me momentarily switching off my settings coz I have format on save enabled and that's messing stuff up already.

This is just a small example of the different preferences we all have and seeing as there are more contributors joining everyone will soon be coming with their own style. 

I, therefore, propose we define our set of code formatting options and et cetera so that we have that consistency in code

**Comment by lekipising** [comment]
I agree with this. We can configure es-lint and prettier to enforce this

**Comment by orama254** [comment]
This is a good suggestion,

I'll get back to this after establishing a preferred file structure. 

The discussion for that is ongoing [HERE](https://github.com/reactdeveloperske/reactdevske-website/discussions/61)
